fake_travis_build.sh

```diff
@@ -23,7 +23,8 @@ targets=("PUBLISHMETA=True" \
     "TARGET=SINGULARITY" \
     "TARGET=SIRINFPV" \
     "TARGET=X_RACERSPI" \
-    "TARGET=RCEXPLORERF3")
+    "TARGET=RCEXPLORERF3"\
+    "TARGET=YUPIF4")
```

Comment:
New targets should not alter the `fake_travis_build.sh` file - this file only contains a subset of the FCs, it is used as a check for PRs, it is not used to generate releases.